test/tools/libtesteth/boostTest.cpp

```diff
@@ -154,19 +154,31 @@ int main( int argc, char* argv[] )
 			ts1->add(BOOST_TEST_CASE(&customTestSuite));
 			framework::master_test_suite().add(ts1);
 		}
+
+		int result;
```

Comment:
Initialize it with zero or something (to be consistent with other parts of the codebase).